frontend/sw360-portlet/src/main/webapp/html/utils/ajax/linkedReleasesAjax.jsp

```diff
@@ -52,8 +53,14 @@
                 <sw360:DisplayEnumOptions type="<%=MainlineState.class%>" selected="${releaseLink.mainlineState}"/>
             </select>
         </td>
-        <td class="deletor">
-            <img src="<%=request.getContextPath()%>/images/Trash.png" onclick="deleteReleaseLink('releaseLinkRow${uuid}','<sw360:out value='${releaseLink.longName}' jsQuoting="true"/>')" alt="Delete">
-        </td>
+        <td class="Action">
+          <sw360:DisplayReleaseLink releaseId="${releaseLink.id}"
+               showName="false" page="<%=PortletReleasePage.EDIT%>"><img src='<%=request.getContextPath()%>/images/edit.png'
+               alt='Edit' title='Edit'>&nbsp;&nbsp;
```

Comment:
Why are the `&nbsp;` contained within the `sw360:DisplayReleaseLink`

Reply:
Instead of `&nbsp;` for styling, it might be better to use CSS margin

Reply:
Thanks @maxhbr  for reviewing the changes.I will change  &nbsp; to css style margin